src/utils/debugutilities.cpp

```diff
@@ -0,0 +1,16 @@
+#include <siminusminus/utils/debugutilities.hpp>
+
+namespace cmm{
+namespace utils{
+
+	void cmm::utils::DebugUtilities::log(const std::string& string, bool newline)
+	{
+		#ifdef _DEBUG
+			std::cout << string << std::endl;
+			if (newline) // We do this to not mix '/n' and std::endl
```

Comment:
then, there isn't any difference about '/n' and std::endl?. I think its better std::endl because not depends on the platform you use.